### DISCUSSION TITLE
🧩 FEATURE REQUEST: Show the URL we're being routed to, not just the relay.

### DISCUSSION THREAD

**Discussion by CodeSpent** [dailydotdev/daily#1003] [Feature Requests]
## Feature Request

Show the URL we're being taken to besides just the daily dev API relay.

### Description

Currently we only see the api.daily.dev URL which leaves us open to navigating to sites we may not want to and depending on how the sourcing for articles is done, could be used maliciously. We should be able to in some capacity see where we're being taken.

**Comment by idoshamun** [comment]
Hi @CodeSpent,

Thank you for the suggestion! 🙏
It's currently not on our roadmap but that might change :)

**Comment by CodeSpent** [comment]
Personally I'd say for the security to time spent, it's definitely a feature worth adding.

It's a risky business having links that mask their destination.
